docs/overview/getting-started.md

```diff
@@ -151,6 +161,17 @@ You can then run this generated binary with:
 subkey
 ```
 
+## Interact with Substrate
+
+The fastest way to interact with any local or public Substrate network is to visit Polkadot-JS Apps:
+
+[https://polkadot.js.org/apps/](https://polkadot.js.org/apps/)
```

Comment:
```suggestion
[https://polkadot.js.org/apps/](https://polkadot.js.org/apps/rpc=ws://localhost:9944)
```

We can automatically point Apps at the local node. However, it is not trivial to interact with a locally running node from Apps because the local node will use an inseucre websocket endpoint.